### PR TITLE
Delete Download History

### PR DESCRIPTION
**Pull request by deeppandya**
Fixes : #1715

### CODE REVIEW

File: chrome/android/java/strings/android_chrome_strings.grd [hunk 1]
```diff
@@ -783,6 +783,9 @@ Your Google account may have other forms of browsing history like searches and a
       <message name="IDS_CLEAR_CACHE_TITLE" desc="Title for Clear Cache in Clear Browsing Data dialog">
         Cached images and files
       </message>
+      <message name="IDS_CLEAR_DOWNLOAD_TITLE" desc="Title for Clear download in Clear Browsing Data dialog">
+        Download history
+      </message>
```
Comment (SergeyZhukovsky) [inline chrome/android/java/strings/android_chrome_strings.grd]: that string has to be added to `strings.xml` as well for translation.

Comment (deeppandya) [reply chrome/android/java/strings/android_chrome_strings.grd]: I checked other menu item texts, and they were not included in strings.xml so i didn't put it there. but now i think it comes from chromium so i must have been included in chromium. i will update it.

File: chrome/android/java/strings/strings.xml [hunk 2]
```diff
@@ -310,4 +310,6 @@ content creators you love.</string>
     <string name="welcome_tour_dialog_text">Relaunching Welcome Tour will not affect any existing settings.</string>
     <string name="finish">Finish</string>
 
+    <string name="clear_download_title"> Download history</string>
```
Comment (SergeyZhukovsky) [inline chrome/android/java/strings/strings.xml]: Shouldn't we remove a space here?

Comment (deeppandya) [reply chrome/android/java/strings/strings.xml]: I added an extra space because this string is not related to onboarding.

**Pull request merged**